tests/generators/TestVectorGenerator.ts

```diff
@@ -0,0 +1,55 @@
+import Jwk from "../../lib/core/versions/latest/util/Jwk";
+import Multihash from "../../lib/core/versions/latest/Multihash";
+import OperationGenerator from "./OperationGenerator";
+
+export default class TestVectorGenerator {
+    /**
+     * Generate and prints out test vectors for operation requests
+     */
+    public static async generateOperationVectors() {
+
+        // generate a create operation request
+        const createOperationData = await OperationGenerator.generateCreateOperation();
```

Comment:
I assume this is random, I suggest you modify your operation generator to be deterministic, see:

https://github.com/transmute-industries/sidetree.js/blob/master/packages/core/src/__test__/test-fixture-generation.test.ts

We generate all fixtures deterministically and compare them too committed values.

Reply:
Good idea. Will make an issue for it. Not going to do it in this pr since the main issue addressing here is SIP 2, it just so happen that fixture has to be updated.

Reply:
https://github.com/decentralized-identity/sidetree/issues/879